DARPA-ASKEM/TERArium
# feat: data explorer and project asset refinements

# Description
Lots of refinements including:
- adding snippets/highlights to data explorer
- adding concept facets for models
- adding human-readable names for items in the side panel for documents and models
- adding related model concepts as tags when expanding a model entry in the explorer
- enabled XDD search to match all/any of the input terms
- add support for filtering XDD using multiple year facet filters

Note this PR bring most of the access to project assets through a separate root-level store variable.

Also note that testing this PR require running the latest "dev" branch of the data service not "main" since it is a bit outdated

Resolves #(issue)

## Type of change


# How Has This Been Tested?
- Open a project and navigate to the side panels of documents and models and note how the listed items now are displayed with title rather than IDs
- Open data explorer and try multiple search terms, e.g. "hospitalizations vaccination" with and without the quotation marks
- Open the data explorer and try a single search term and note the snippets/highlights now included in the search results
- Open the data explorer and switch to the "Models" tab and note a new facet about concepts. You can also expand any model entry to see the list of concept tags

# Checklist:
